diff -urNad OpenJPEG_v1_3-orig/Makefile OpenJPEG_v1_3/Makefile
--- OpenJPEG_v1_3-orig/Makefile	2007-12-21 12:39:41.000000000 +0200
+++ OpenJPEG_v1_3/Makefile	2010-10-04 00:43:14.000000000 +0300
@@ -18,11 +18,11 @@
 # Converts cr/lf to just lf
 DOS2UNIX = dos2unix
 
-COMPILERFLAGS = -Wall -O3 -ffast-math -std=c99 -fPIC
+COMPILERFLAGS = -Wall -ffast-math -std=c99 -fPIC
 LIBRARIES = -lstdc++
 
 MODULES = $(SRCS:.c=.o)
-CFLAGS = $(COMPILERFLAGS) $(INCLUDE)
+CFLAGS = $(COMPILERFLAGS) $(MYCFLAGS) $(INCLUDE)
 
 TARGET  = openjpeg
 STATICLIB = lib$(TARGET).a
